src/control/jobs/control_jobs.c

```diff
@@ -368,9 +368,18 @@ static int dt_control_merge_hdr_process(dt_imageio_module_data_t *datai, const c
     d->orientation = image.orientation;
     for(int i = 0; i < 3; i++)
       d->wb_coeffs[i] = image.wb_coeffs[i];
-    for(int k=0; k<4; k++)
-      for(int i=0; i<3; i++)
-        d->adobe_XYZ_to_CAM[k][i] = image.adobe_XYZ_to_CAM[k][i];
+    // give priority to DNG embedded matrix, see dt_colorspaces_conversion_matrices_xyz()
+    if(!isnan(image.d65_color_matrix[0]))
```

Comment:
// and the corresponding call in temperature.c passing 
// image_storage.d65_color_matrix in in_XYZ_to_CAM[9].

Maybe worth adding? At least it was not straightforward to understand this for me.

Reply:
Done